### PR TITLE
Carlos/fix fbopen full importer

### PR DESCRIPTION
**Pull request by calumhalcrow**

### CODE REVIEW

**Comment by calumhalcrow** [comment]
@carloslima what's the root cause of the bug?

File: app/importers/trade_lead/fbopen_importer/full_data.rb [hunk 1]
```diff
@@ -62,7 +72,7 @@ def process_xml_entry(entry)
         return nil if !(entry[:specific_location].try(:upcase) =~ /\A[A-Z]{2}\z/) || entry[:specific_location] == 'US'
 
         entry = process_xml_dates(entry)
-        return nil unless entry[:end_date] >= Date.today || entry[:end_date].nil?
+        return nil unless entry[:end_date].nil? || entry[:end_date] >= Date.today
```
Comment (carloslima) [inline app/importers/trade_lead/fbopen_importer/full_data.rb]: @calumhalcrow This ^^
Wrong ordering.

Comment (calumhalcrow) [inline app/importers/trade_lead/fbopen_importer/full_data.rb]: Heh, yeah that would be a problem if `entry[:end_date]` was nil :laughing:

**Comment by calumhalcrow** [comment]
You've imporved our CodeClimate score by 0.06 :+1: 

https://codeclimate.com/github/GovWizely/webservices/compare/carlos/fix-fbopen-full-importer

**Comment by calumhalcrow** [comment]
Coverage is < 100% though. The case where `entries.size >= 1000` isn't covered.

**Comment by carloslima** [comment]
Should be better now :-)

**Pull request merged**